### PR TITLE
Consistent use of POINTER_TYPE to read/write pointer values.

### PR DESCRIPTION
**Pull request by sbc100**
We could go the other way and use `void*` everywhere?

### CODE REVIEW

**Comment by dschuff** [comment]
`makeSetValue`/`setValue` just generate JS code that writes through `HEAP32` or whatever, right? So whether we call it `POINTER_TYPE` or `void*` is mostly a stylistic question?

**Comment by sbc100** [comment]
> `makeSetValue`/`setValue` just generate JS code that writes through `HEAP32` or whatever, right? So whether we call it `POINTER_TYPE` or `void*` is mostly a stylistic question?

Indeed, all types that end if `*` will end up treated as `POINTER_TYPE` anyway... so its just style.

File: src/library_glfw.js [hunk 1]
```diff
@@ -780,7 +780,7 @@ var LibraryGLFW = {
 
         var filename = allocateUTF8(path);
         filenamesArray.push(filename);
-        setValue(filenames + i*4, filename, 'i8*');
+        setValue(filenames + i*4, filename, '{{{ POINTER_TYPE }}}');
```
Comment (dschuff) [inline src/library_glfw.js:783]: I suppose one advantage of `i8*` or `void*` would be that you don't need the `{{{ }}}` here. Not sure how often that happens though, it looks like this is the only place in this PR.

Comment (sbc100) [reply src/library_glfw.js:783]: We should remove the final uses of setValue.. so it should no longer exist: See #16849

Comment (dschuff) [reply src/library_glfw.js:783]: LGTM. I guess it doesn't matter much then. POINTER_TYPE seems more obvious just from reading than `void*` so this seems fine as it is.

**Review by dschuff — APPROVED** [review]

**Comment by sbc100** [comment]
@kripken WDYT?  Do you have a preference?

**Pull request merged**